UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -159,9 +185,12 @@ public async Task<bool> CreateCatalogFile(string batchId, string exchangeSetRoot
 
             if (!string.IsNullOrWhiteSpace(exchangeSetRootPath))
             {
+                DateTime createCatalogueFileTaskStartAt = DateTime.UtcNow;
                 logger.LogInformation(EventIds.CreateCatalogFileRequestStart.ToEventId(), "Create catalog file request started for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", batchId, correlationId);
                 isFileCreated = await fulfilmentAncillaryFiles.CreateCatalogFile(batchId, exchangeSetRootPath, correlationId, listFulfilmentData, salesCatalogueDataResponse);
                 logger.LogInformation(EventIds.CreateCatalogFileRequestCompleted.ToEventId(), "Create catalog file request completed for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", batchId, correlationId);
+                DateTime createCatalogueFileTaskCompletedAt = DateTime.UtcNow;
+                monitorHelper.MonitorRequest("Create Catalogue File Task", createCatalogueFileTaskStartAt, createCatalogueFileTaskCompletedAt, correlationId,null,null, null,batchId);
```

Comment:
Kindly rename Create Catalogue File Task to "Create Catalog File Task" and createCatalogueFileTaskStartAt to createCatalogFileTaskStartedAt and createCatalogueFileTaskCompletedAt to createCatalogFileTaskCompletedAt

Reply:
Done